examples/vello_editor/src/main.rs

```diff
@@ -215,6 +219,20 @@ impl ApplicationHandler<accesskit_winit::Event> for SimpleVelloApp<'_> {
         self.editor.handle_event(event.clone());
         if self.last_drawn_generation != self.editor.generation() {
             render_state.window.request_redraw();
+            let area = self.editor.editor().ime_cursor_area();
+            if self.last_sent_ime_cursor_area != Some(area) {
+                self.last_sent_ime_cursor_area = Some(area);
+                // Note: on X11 `set_ime_cursor_area` may cause the exclusion area to be obscured
+                // until https://github.com/rust-windowing/winit/pull/3966 is in the Winit release
+                // used by this example.
+                render_state.window.set_ime_cursor_area(
+                    PhysicalPosition::new(
```

Comment:
Hmm, is physical right here?

Reply:
I think it is, but I haven't tested it with a non-`1` display scale. The editor gets its width set as the physical width in https://github.com/linebender/parley/blob/85a824a5310781ef5f3b9568f95e0e613cdcc528/examples/vello_editor/src/main.rs#L228-L235